deprecated/packages/transifex-client/transifex-client.nuspec

```diff
@@ -18,18 +18,11 @@
     <tags>transifex-client transifex translation cli cross-platform foss</tags>
     <summary>The Transifex Command-line Tool enables you to manage your translations within a project without the need of an elaborate UI system.</summary>
     <!-- Do not touch the description here in the nuspec file. Description is imported during update from the Readme.md file -->
-    <description><![CDATA[The Transifex Command-line Tool enables you to manage your translations within a project without the need of an elaborate UI system.
-
-You can use the command line tool to create new resources, map locale files to translations, and synchronize your Transifex project with your local repository. Translators and localization managers can use it to handle large volumes of translation files. The Transifex Command-line Tool can help to enable continuous integration workflows and can be run from CI servers like Jenkins and Bamboo.
-
-# Notes
-- This package uses the Python 3.x version of the transifex-client
-- This package only provides the 64-bit executable due to a number of positives on VirusTotal.
-  The Software Authors believe them to be false positives, but I as the package maintainer am not convinced.
-]]></description>
+    <description><![CDATA[This package is deprecated as the upstream maintainer has deprecated the Python verson of the Transifex client along with the API endpoints it used.
+    Please use the replacement transifex-cli package going forward.]]></description>
```

Comment:
Just a suggestion based on what we have done for a previous package we deprecated.

```suggestion
    <description><![CDATA[### New package name: **[transifex-cli](https://community.chocolatey.org/packages/transifex-cli)**
    
--

`transifex-client` has been deprecated as the upstream maintainer has deprecated the Python version of the Transifex client along with the API endpoints it used.

It exists only for backward compatibility to provide a transitional dummy package for existing users and scripts which may still use this package ID.]]></description>
```

Reply:
Updated this one too